### ISSUE TITLE
Polish translation fix

### ISSUE THREAD

**Issue by wojciechrak** [jderose9/dash-to-panel#327]
I've noticed one problem with polish translation.

https://github.com/jderose9/dash-to-panel/blob/223d0152510adb460c5762193302a03ad023656f/po/pl.po#L295-L301

**Left of status menu**
![left_of_status_menu](https://user-images.githubusercontent.com/3034230/35748901-e23692ea-084f-11e8-842e-259b6ead66cc.png)

**Right of status menu**
![right_of_status_menu](https://user-images.githubusercontent.com/3034230/35748902-e25223ac-084f-11e8-878b-b6974abbb2e4.png)

Texts should be swapped like this:
```
#: Settings.ui.h:55
msgid "Left of status menu"
msgstr "Z lewej strony menu systemowego"

#: Settings.ui.h:56
msgid "Right of status menu"
msgstr "Z prawej strony menu systemowego"
```

**Comment by pkomur** [comment]
Fixed with pull request #323

**Comment by jderose9** [comment]
Thanks to you both!
